client/src/pages/ScuntJudgeForm/scuntTempData.jsx

```diff
@@ -0,0 +1,68 @@
+﻿export const list = [
+  {
+    id: '1',
```

Comment:
For this list, can we switch `id` to be something else? (Anything else would be fine) - we will run into issues with this + mongoose which we are running on the backend